critiquebrainz/db/users.py

```diff
@@ -0,0 +1,17 @@
+from critiquebrainz import db
+from critiquebrainz.db import exceptions as db_exceptions
+import sqlalchemy
+
+
+def getusers_id(usernames):
+    """Get MusicBrainz ids of users
+    Args:
+        usernames (list): MusicBrainz usernames.
+    """
+    with db.engine.connect() as connection:
+        req = sqlalchemy.text("""SELECT * FROM "user" WHERE display_name IN :users""")
+        result = connection.execute(req, users=tuple(usernames))
```

Comment:
Minor thing, but it would be better to keep all queries defined consistently:

```python
result = connection.execute("""
    SELECT *
      FROM "user"
     WHERE display_name IN :users
""", {"users": tuple(usernames)})
```

See https://github.com/metabrainz/critiquebrainz/blob/master/critiquebrainz/db/vote.py#L69-L75 for example.